test/functional/wallet_migration.py

```diff
@@ -529,11 +529,20 @@ def test_default_wallet(self):
         self.log.info("Test migration of the wallet named as the empty string")
         wallet = self.create_legacy_wallet("")
 
-        self.migrate_wallet(wallet)
+        # Move time to have a unique name
+        curr_time = int(time.time()) + 100
+        wallet.setmocktime(curr_time)
+
+        res = self.migrate_wallet(wallet)
```

Comment:
Good idea on putting the time into the name, but why do you add 100 instead of just using the current time?

Reply:
> why do you add 100 instead of just using the current time?

Just because I coded it while we were on the weekly meeting.. and took `time.time()` as the node's mocked time. Fixed. Thanks.